corehq/apps/users/static/users/js/custom_data_fields.js

```diff
@@ -36,7 +36,7 @@ hqDefine("users/js/custom_data_fields", [
         var originalProfileFields = {},
             originalProfileId,
             originalProfile;
-        if (options.user_data) {
+        if (Object.keys(options.user_data).length) {
```

Comment:
should this be just changed back to `options.user_data`

Reply:
js doesn't read empty dicts as false which is what I assume whoever wrote that first conditional was trying to check for so this makes more sense

Reply:
oh right, I'm getting the "truthy" value of an object confused with how it works in python. Nice fix!